WordPress/Classes/Services/ReaderPostService.m

```diff
@@ -1227,6 +1227,12 @@ - (ReaderPost *)createOrReplaceFromRemotePost:(RemoteReaderPost *)remotePost for
         post.sourceAttribution = nil;
     }
 
+    if ([Feature enabled:FeatureFlagReaderWebview]) {
+        post.content = remotePost.content;
+    } else {
+        post.content = [RichContentFormatter formatContentString:remotePost.content isPrivateSite:remotePost.isBlogPrivate];
+    }
+
```

Comment:
Looks good, but I think we'd want to either remove the earlier assignment on line 1156, or we'd want to replace it with this snippet.

Reply:
Fixed in 0d64166f0ee91b53c503227ee4f3f58ce812f064